### PR TITLE
chore(dependencies): Bump ember-elsewhere

### PR DESCRIPTION
**Pull request by srowhani**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

https://github.com/ef4/ember-elsewhere/pull/15

This is to address teardown issues for acceptance tests

# CHANGELOG
- Bumped ember-elsewhere version

### CODE REVIEW

**Comment by job13er** [comment]
Do we want to do something about bitHound? If we don't care about failures in it, I'd much rather disable it. I don't like the idea of us getting comfortable merging PRs with red Xs in them, it makes it far more probably that a legitimate error gets merged in.

**Comment by sglanzer-fire** [comment]
Bithound feels like a lot of noise currently, I'd be fine with disabling it

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/436/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

File: blueprints/ember-frost-core/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
     const addonsToAdd = [
       {name: 'ember-computed-decorators', target: '~0.2.0'},
       {name: 'ember-concurrency', target: '~0.7.15'},
-      {name: 'ember-elsewhere', target: '~0.4.1'},
+      {name: 'ember-elsewhere', target: '~1.0.1'},
```
Comment (sglanzer-fire) [inline blueprints/ember-frost-core/index.js:8]: No breaking changes I take it?

**Pull request merged**